config/locales/de.yml

```diff
@@ -352,7 +353,7 @@ de:
         role: Rolle
       volunteer: &id-volunteer_attributes
         <<: [*id-availability, *id-generic_keys, *id-volunteer-acceptance]
-        acceptance: Affirmation
+        acceptance: *id-volunteer-acceptance
```

Comment:
If `acceptance: Affirmation` gets removed then we'll have Acceptance again in the filter and the form. I also thought that this line was covered by this one: `<<: [*id-availability, *id-generic_keys, *id-volunteer-acceptance]`.

Reply:
True